plugins/modules/rabbitmq_user.py

```diff
@@ -461,7 +838,13 @@ def main():
         force=dict(default='no', type='bool'),
         state=dict(default='present', choices=['present', 'absent']),
         node=dict(default='rabbit'),
-        update_password=dict(default='on_create', choices=['on_create', 'always'], no_log=False)
+        update_password=dict(default='on_create', choices=['on_create', 'always'], no_log=False),
+        login_protocol=dict(type="str", default="http", choices=["http", "https"],
+                            aliases=['protocol'], no_log=False),
+        login_host=dict(type="str", aliases=['host'], default=None),
+        login_port=dict(type="str", aliases=['port'], default=None),
+        login_user=dict(type="str", default=None, no_log=True),
+        login_password=dict(type="str", default=None, no_log=True)
```

Comment:
```suggestion
        login_user=dict(type="str", no_log=True),
        login_password=dict(type="str", no_log=True)
```
as far as I remember it's `None` by default for str parameters